CODE_REVIEW.md

```diff
@@ -0,0 +1,26 @@
+Code Smells Found:
```

Comment:
Excellent, thorough code review.

Reply:
Thank you :-) 
Will fix below review comments and update the PR. I followed the Metro Web Confluence to fix accessibility here which is wrong move.

Reply:
Oh good to know, yeah there may be some bad information in there. I'll ask ARC to check that out.